### PR TITLE
Feature/command groups bits

### PR DESCRIPTION
**Pull request by vichik123**

### CODE REVIEW

File: src/main/java/frc/robot/commandgroups/bits/TestBallFlow.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        addCommands(
+                pickUpCargo.withInterrupt(pickUpCargoIsFinished),
+                new WaitCommand(5),
+                shootCargo.withInterrupt(shootCargoIsFinished)
+        );
+    }
+
+    @Override
+    public void execute() {
+        if (pickUpCargo.isScheduled()) {
```
Comment (barel13) [inline src/main/java/frc/robot/commandgroups/bits/TestBallFlow.java:43]: I think you need to call `super.execute()` in order to call the functions from the command group

File: src/main/java/frc/robot/subsystems/conveyor/Conveyor.java [hunk 2]
```diff
@@ -118,6 +118,10 @@ public void setPower(double power) {
         motor.set(power);
     }
 
+    public double getPower() {
```
Comment (barel13) [inline src/main/java/frc/robot/subsystems/conveyor/Conveyor.java]: docs

File: src/main/java/frc/robot/subsystems/intake/Intake.java [hunk 3]
```diff
@@ -43,6 +43,10 @@ public void setPower(double power) {
         motor.set(ControlMode.PercentOutput, power);
     }
 
+    public double getPower() {
```
Comment (barel13) [inline src/main/java/frc/robot/subsystems/intake/Intake.java]: You know what I'm gonna write...

**Review by barel13 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commandgroups/bits/TestBallFlow.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+public class TestBallFlow extends SequentialCommandGroup {
+    private final Intake intake;
+    private final Shooter shooter;
+    private final Conveyor conveyor;
+    private final ShootCargo shootCargo;
+    private final PickUpCargo pickUpCargo;
+
+    public TestBallFlow(Conveyor conveyor, Intake intake, Shooter shooter,
+                        PickUpCargo pickUpCargo, BooleanSupplier pickUpCargoIsFinished,
+                        ShootCargo shootCargo, BooleanSupplier shootCargoIsFinished) {
```
Comment (barel13) [inline src/main/java/frc/robot/commandgroups/bits/TestBallFlow.java]: You don't need to pass commands, just their parameters, and then create them internally

**Review by barel13 — CHANGES_REQUESTED** [review]

**Review by barel13 — APPROVED** [review]

**Review by adamv17 — APPROVED** [review]

**Pull request merged**